server/index.ts

```diff
@@ -26,7 +26,11 @@ api.use("/*", cors(), logger(), prettyJSON());
 
 appointments.get("/upcoming", async (c) => {
   try {
-    const appointments = await findboligClient.getUpcomingAppointments();
+    const cookies = c.req.header("x-findbolig-cookies");
```

Comment:
This code should be refactored to a reusable function.
Create a new file in `/lib/`